src/components/Rankings/Rankings.js

```diff
@@ -0,0 +1,59 @@
+import "./Ranking.css";
+
+const Rankings = ({ puns, devs }) => {
+  const punsRanking = puns.sort((pun, anotherPun) => {
+    return -pun.votes + anotherPun.votes;
+  });
+  const devsRanking = devs.sort((dev, anotherDev) => {
+    return -dev.puns + anotherDev.puns;
```

Comment:
```suggestion
    return  anotherDev.puns - dev.puns;
```
This isn't the same thing?